cognite/client/data_classes/simulators/runs.py

```diff
@@ -129,19 +159,35 @@ def __init__(
     @classmethod
     def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None) -> SimulationRunWrite:
         inputs = resource.get("inputs", None)
+        routine_revision_external_id = resource.get("routineRevisionExternalId", None)
+        model_revision_external_id = resource.get("modelRevisionExternalId", None)
+        routine_external_id = resource.get("routineExternalId", None)
```

Comment:
None is default so no need to specify
```suggestion
        routine_revision_external_id = resource.get("routineRevisionExternalId")
        model_revision_external_id = resource.get("modelRevisionExternalId")
        routine_external_id = resource.get("routineExternalId")
```

Reply:
fixed! :)